tests/FieldAttributesValidationTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Styde\Html\Tests;
+
+use Styde\Html\Facades\Field;
+use Illuminate\Validation\Rules\Exists;
+
+class FieldAttributesValidationTest extends TestCase
+{
+    /** @test */
+    function it_validate_that_the_required_attribute_generates_the_required_rule()
+    {
+        $field = Field::text('name', ['required' => true]);
+
+        $this->assertSame(['required'], $field->getValidationRules());
+    }
+
+    /** @test */
+    function it_not_generate_the_required_rule_if_the_attribute_is_not_present()
+    {
+        $field = Field::text('name');
+
+        $this->assertNotSame(['required'], $field->getValidationRules());
+    }
+
+    /** @test */
+    function it_validate_the_existence_of_the_nullable_rule()
+    {
+        $field = Field::text('name');
+
+        $this->assertSame(['nullable'], $field->getValidationRules());
+    }
+
+    /** @test */
+    function it_validate_the_existence_of_the_email_rule()
+    {
+        $field = Field::email('email', ['required' => true]);
+
+        $this->assertEquals(['required', 'email'], $field->getValidationRules());
+    }
+
+    /** @test */
+    function it_validate_that_the_email_rule_does_not_exist()
+    {
+        $field = Field::text('name', ['required' => false]);
+
+        $this->assertNotEquals(['email'], $field->getValidationRules());
+    }
+
+    /** @test */
+    function it_validate_the_existence_of_the_url_rule()
+    {
+        $field = Field::url('link', ['required' => true]);
+
+        $this->assertEquals(['required', 'url'], $field->getValidationRules());
+    }
+
+    /** @test */
+    function it_validate_that_the_url_rule_does_not_exist()
+    {
+        $field = Field::text('name', ['required' => true]);
+
+        $this->assertNotEquals(['required', 'url'], $field->getValidationRules());
+    }
+
+    /** @test */
+    function it_there_are_multiples_rules()
+    {
+        $field = Field::email('email', ['required']);
+
+        $this->assertEquals(['required', 'email'], $field->getValidationRules());
+    }
+
+    /** @test */
+    function it_select_field_has_the_required_rule_and_options()
```

Comment:
should only test the options here, there's another test for multiple rules above